packages/shared/components/popups/Diagnostics.svelte

```diff
@@ -1,65 +1,65 @@
 <script lang="typescript">
-    import { Button, Text } from 'shared/components'
     import { appSettings, appVersionDetails } from '@core/app'
-    import { Platform } from 'shared/lib/platform'
+    import { localize } from '@core/i18n'
     import { activeProfile } from '@core/profile'
+    import { Button, Text } from 'shared/components'
+    import { Platform } from 'shared/lib/platform'
     import { setClipboard } from 'shared/lib/utils'
-    import { Locale } from '@core/i18n'
+    import { onMount } from 'svelte'
 
-    export let locale: Locale
+    const { loggedIn } = $activeProfile ?? {}
 
     let contentApp = ''
     let contentSystem = ''
 
-    const combineValues = (values) =>
-        values.map((c) => (c.label ? `${locale(c.label)}: ${c.value}` : c.value)).join('\r\n')
-
-    const appVars = [
-        {
-            label: '',
-            value: locale('views.dashboard.security.version.title', {
-                values: { version: $appVersionDetails.currentVersion },
-            }),
-        },
-    ]
-
-    if ($activeProfile) {
-        appVars.push({
-            label: 'views.settings.language.title',
-            value: $appSettings.language,
-        })
-        appVars.push({
-            label: 'views.settings.currency.title',
-            value: $activeProfile?.settings.currency,
-        })
-        appVars.push({
-            label: 'views.settings.networkConfiguration.nodeConfiguration.title',
-            value: locale(
-                `views.settings.networkConfiguration.nodeConfiguration.${
-                    $activeProfile?.settings.clientOptions.automaticNodeSelection ? 'automatic' : 'manual'
-                }`
-            ),
-        })
-        appVars.push({
-            label: 'general.nodeList',
-            value: $activeProfile?.settings.clientOptions.nodes.map((node) => node?.url).toString(),
-        })
-    }
-
-    contentApp = combineValues(appVars)
+    const combineValues = (values): string =>
```

Comment:
I think this could be clearer
```suggestion
    const concatenateInfo = (values: { label?: string, value: string | number }[] ): string =>
```